src/components/PrimaryNav/PrimaryNav.jsx

```diff
@@ -63,15 +63,17 @@ NavHeader.propTypes = {
     headerLogo: PropTypes.node
 };
 
-export const PrimaryNav = ({headerLogo, top, bottom, headerCaption, modeIcon, ...props}) => {
+export const PrimaryNav = ({headerLogo, top, bottom, headerCaption, modeIcon, isCollapsibleOnCLick, ...props}) => {
```

Comment:
Do we want to keep the choice collapsible on click or not?
I think we can force the collapse in all cases.

If we keep it, please fix the typo `isCollapsibleOnClick`

Reply:
My original idea was to make this behaviour optional but seeing how contested it is and that there is no clear direction for what  should be done and given that not having it means that I won't have to modify any other module I decided to remove it :)